engine/src/flutter/lib/web_ui/lib/src/engine/skwasm/skwasm_impl/renderer.dart

```diff
@@ -20,6 +20,12 @@ class SkwasmRenderer extends Renderer {
 
   SkwasmPathConstructors pathConstructors = SkwasmPathConstructors();
 
+  @override
+  set resourceCacheMaxBytes(int bytes) {
+    // ?
+    // rasterizer.setResourceCacheMaxBytes(bytes);
+  }
```

Comment:
@eyebrowsoffire @harryterkelsen should we do this in Skwasm? I don't see any usages of this in the framework, so I'm not sure who uses it.

Reply:
It is documented in the framework as something that can be called here: 

https://github.com/flutter/flutter/blob/fdab2850235b4a3a5ef1447f002a34902329a596/packages/flutter/lib/src/services/system_channels.dart#L417

I added code to enable this in Skwasm as part of the unification, but forgot to plumb it all the way through. I think for consistency, we should add this here. I also don't know who actually uses this but it's even enabled in the mobile engine.